### PR TITLE
Minor fix typo

### PR DESCRIPTION
**Pull request by ScorpioCPH**
Minor fix comments typo.

### CODE REVIEW

File: tensorflow/core/lib/gtl/array_slice.h [hunk 1]
```diff
@@ -191,8 +191,8 @@ class ArraySlice {
   void pop_front() { remove_prefix(1); }
 
   // These relational operators have the same semantics as the
-  // std::vector<T> relational operators: they do deep (elementwise)
-  // comparisons.  Array slices are equal iff their size is the same
```
Comment (caisq) [inline tensorflow/core/lib/gtl/array_slice.h]: This is not a typo. "Iff" means "if and only if".
https://www.merriam-webster.com/dictionary/iff

File: tensorflow/core/protobuf/worker.proto [hunk 2]
```diff
@@ -86,7 +86,7 @@ message RegisterGraphRequest {
   // having its device_name filled in.
   GraphDef graph_def = 2;
 
-  // True iff the graph (before partitioning) contains control flow nodes.
```
Comment (caisq) [inline tensorflow/core/protobuf/worker.proto]: Same as above.

**Review by caisq — CHANGES_REQUESTED** [review]

File: tensorflow/core/lib/gtl/array_slice.h (hunk 1 above)
Comment (ScorpioCPH) [inline tensorflow/core/lib/gtl/array_slice.h]: ok, i will revert it.

**Comment by ScorpioCPH** [comment]
@caisq ping.

File: tensorflow/core/lib/gtl/array_slice.h [hunk 3]
```diff
@@ -191,8 +191,8 @@ class ArraySlice {
   void pop_front() { remove_prefix(1); }
 
   // These relational operators have the same semantics as the
-  // std::vector<T> relational operators: they do deep (elementwise)
   // comparisons.  Array slices are equal iff their size is the same
+  // std::vector<T> relational operators: they do deep (element-wise)
```
Comment (caisq) [inline tensorflow/core/lib/gtl/array_slice.h]: I think this line is moved to the wrong place.

**Review by caisq — CHANGES_REQUESTED** [review]

File: tensorflow/core/lib/gtl/array_slice.h (hunk 3 above)
Comment (ScorpioCPH) [inline tensorflow/core/lib/gtl/array_slice.h]: You are right, fix it by amend last commit :)

**Review by caisq — APPROVED** [review]

**Pull request merged**